lib/pure/random.nim

```diff
@@ -213,15 +213,27 @@ proc rand*(r: var Rand; max: Natural): int {.benign.} =
   ## * `rand proc<#rand,Rand,HSlice[T: Ordinal or float or float32 or float64,T: Ordinal or float or float32 or float64]>`_
   ##   that accepts a slice
   ## * `rand proc<#rand,typedesc[T]>`_ that accepts an integer or range type
-  runnableExamples("-r:off"):
+  runnableExamples:
     var r = initRand(123)
-    assert r.rand(100) == 96 # implementation defined
-
+    if false:
+      assert r.rand(100'u) == 96'u # implementation defined
+  # pending csources >= 1.4.0 or fixing https://github.com/timotheecour/Nim/issues/251#issuecomment-831599772,
+  # use `runnableExamples("-r:off")` instead of `if false`
```

Comment:
No, not "pending csources >= 1.4.0", stop pushing your agenda. The csources work just fine and if the compiler cannot bootstrap without constant csources updates, maybe it's because it has too many dependencies?

Reply:
fixed for sake of moving on with this PR but it will have to happen at some point or another (eg around time 2.0 is released), at which point it's only a good idea to keep track of places needed fixing.